src/utils_general/statistics.py

```diff
@@ -109,3 +111,38 @@ def calc_mpe(observations: np.array, forecast: np.array) -> float:
         / len(observations.time)
         * 100
     )
+
+
+def calc_crps(
+    observations: xr.DataArray,
+    forecasts: xr.DataArray,
```

Comment:
this is the only place where xarray is used. @turnerm Do you know what is the best practice when only using a package to define the type?

Reply:
I think it's totally legit!